independent-publisher-connectors/Yelp/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1238 @@
+            "x-ms-summary": "Limit",
+            "x-ms-visibility": "advanced",
+            "format": "int32"
+          },
+          {
+            "name": "match_threshold",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Specifies whether a match quality threshold should be applied to the matched businesses. Must be one of 'none', 'default' or 'strict'. none: Do not apply any match quality threshold; all potential business matches will be returned. default: Apply a match quality threshold such that only very closely matching businesses will be returned. strict: Apply a very strict match quality threshold.",
+            "x-ms-summary": "Match Threshold",
+            "x-ms-visibility": "advanced",
+            "format": "int32",
+            "enum": [
+              "none",
+              "default",
+              "strict"
+            ]
+          }
+        ]
+      }
+    },
+    "/v3/businesses/{id}/reviews": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "reviews": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "rating": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "rating"
+                      },
+                      "user": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "string",
+                            "description": "id"
+                          },
+                          "profile_url": {
+                            "type": "string",
+                            "description": "profile_url"
+                          },
+                          "image_url": {
+                            "type": "string",
+                            "description": "image_url"
+                          },
+                          "name": {
+                            "type": "string",
+                            "description": "name"
+                          }
+                        },
+                        "description": "user"
+                      },
+                      "text": {
+                        "type": "string",
+                        "description": "text"
+                      },
+                      "time_created": {
+                        "type": "string",
+                        "description": "time_created"
```

Comment:
@ahmadn82 All the description values of the properties are expected to be free from grammatical and spelling errors. Can you please provide valid descriptions.

Reply:
@sriyen-msft Fixed!